bryant-bryantbuilt/week-3/giffaw/scripts/app.js

```diff
@@ -0,0 +1,33 @@
+console.log('App.js connected');
+$(document).ready(function(){
+  console.log('jQuery ready');
+  let giphyUrl = "http://api.giphy.com/v1/gifs/search?q=";
+
+  let apiKey = "5Nm2vT5hbEHgwBReVHBEQJ4pA2GXPX7J";
```

Comment:
Like that you added the api key here, but it was provided in the form inside of the html. It is important to read through all the provided code to reduce down on repetitive code. This will be a good habit once you are in a real work environment.